test/unit_Interval.jl

```diff
@@ -120,4 +120,13 @@ for N in [Float64, Float32, Rational{Int}]
     @test dAB == Interval(N(5), N(6))
     @test dAC == Interval(N(5), N(8))
     @test dAD == UnionSet(Interval(N(5), N(6)), Interval(N(7), N(8)))
+
+    # check if an interval is flat, i.e. if its endpoints coincide (to numerical precision)
+    ztol = LazySets.ABSZTOL(N) # pick up default absolute zero tolerance value
```

Comment:
Should this not be a constant value? If we change `ABSZTOL` (see #1341), this test may break.

Reply:
Hmm i don't think that it will break for another value, right? Precisely because `isflat(Interval(N(0), x))` holds for all `x <= ztol`, since it uses `isapproxzero`, while it  doesn't hold for bigger ones (eg. `2ztol`).

Reply:
I see. That makes this test depend on the implementation, but I do not see a better way :+1: